docker/librom/Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN sudo git clone https://github.com/LLNL/libROM.git
 WORKDIR ./libROM
 RUN sudo git pull
 # pylibROM is currently based on a specific commit of the libROM.
-RUN sudo git checkout mfem_fix
+RUN sudo git checkout 0809d7d09dc24f0963c38fc8c0a2649948142ba0

```

Comment:
It looks like this change was actually part of #17, but should this be changed to be the head of the main branch of libROM? This commit SHA seems to be outdated since a few PRs have been merged in libROM. Maybe this should be changed in a separate PR to make sure the two repos stay in sync.

Reply:
I guess we are using a static commit for libROM to keep pylibROM stable and avoid any unpredicted breakdowns due to the actively changing C++ libROM. From time to time I saw Kevin change the SHA, after some necessary changes were made in C++ libROM. @dreamer2368 , may I ask if it is accurate?

Reply:
This is correct. We'd like to update SHA occasionally, but at the same time specify SHA explicitly which can be information for debugging. I think `mfem_fix` branch is now merged to main branch of the libROM, so it's okay to update this SHA now.